task/k8s/resources/resource_job.go

```diff
@@ -62,13 +62,13 @@ func (j *Job) Create(ctx context.Context) error {
 		"m":       "8-32000",
 		"l":       "32-128000",
 		"xl":      "64-256000",
-		"m+t4":    "4-16000+nvidia-tesla-t4*1",
-		"m+k80":   "4-64000+nvidia-tesla-k80*1",
-		"l+k80":   "32-512000+nvidia-tesla-k80*8",
-		"xl+k80":  "64-768000+nvidia-tesla-k80*16",
-		"m+v100":  "8-64000+nvidia-tesla-v100*1",
-		"l+v100":  "32-256000+nvidia-tesla-v100*4",
-		"xl+v100": "64-512000+nvidia-tesla-v100*8",
+		"m+t4":    "4-16000+nvidia*1",
+		"m+k80":   "4-64000+nvidia*1",
+		"l+k80":   "32-512000+nvidia*8",
+		"xl+k80":  "64-768000+nvidia*16",
+		"m+v100":  "8-64000+nvidia*1",
+		"l+v100":  "32-256000+nvidia*4",
+		"xl+v100": "64-512000+nvidia*8",
```

Comment:
temporary & needs to be reverted?

Reply:
Unfortunately not temporary, and definitely not to be reverted. 😞 Read the pull request body for details on this tradeoff.

Reply:
We may want to use `affinity` in a near future, but it's better to make it work out of the box instead of asking users to label their nodes manually.

Reply:
ok follow-up issue then

Reply:
Yes

Reply:
Moved to https://github.com/iterative/terraform-provider-iterative/issues/158#issuecomment-1111267672